Misc/NEWS.d/next/Library/2022-04-08-08-55-36.bpo-47087.Q5C3EI.rst

```diff
@@ -0,0 +1,2 @@
+Implement ``typing.Required`` and ``typing.NotRequired`` (:pep:`655`). Patch
+by Jelle Zijlstra.
```

Comment:
```suggestion
by David Foster and Jelle Zijlstra.
```
@davidfstr wrote the original code in typing-extensions so I shouldn't claim sole credit.

Reply:
> @davidfstr wrote the original code in typing-extensions so I shouldn't claim sole credit.

Then also add a Contributed-by header (or whatever it's called) to the commit message?

Reply:
I put it in the PR header and I'll add it to the commit message when I merge this. (I'll give it some more time though, to give other people a chance to give feedback.)